### DISCUSSION TITLE
Potential StarRocks integration

### DISCUSSION THREAD

**Discussion by bugrakilic** [getredash/redash#7008] [Q&A]
Hello, as you all know Redash has many integrations with well-known databases and data warehouse solutions. Is there also a way to integrate Redash with [StarRocks](https://www.starrocks.io)? Or is there any plan to feature it? Thanks in advance!

**Comment by guykatz** [comment]
i am in terested in this as well.
does mysql support means starrocks support? (as its build on it as at its core)
10x
